packages/gatsby-recipes/src/index.js

```diff
@@ -1,4 +1,7 @@
-module.exports = recipe => {
+export { startGraphQLServer } from "./graphql-server"
+
+// data = { recipe?: string, graphqlPort: number, projectRoot: string }
+export default async data => {
```

Comment:
This is great, but could you turn this into a named function rather than an anonymous arrow. You can still make it a default export, but it means it's easier to profile. This is a request from @pvdz for all exports to be named.